3.Tests/UI/StepDefinitions/CommonStepDefinition.cs

```diff
@@ -237,7 +243,7 @@ public void VerifyAlertMessage(string message)
         Assert.That(message, Is.EqualTo(alert.Text));
     }
 
-    [When(@"the user accepts the alert")]
+    [When(@"(?:the user )?accepts the alert")]
     public void AcceptAlert()
     {
         GenericWebDriver.Instance.SwitchTo().Alert().Accept();
```

Comment:
You could create a method on the GenericWebDriver singleton to accept an alert, it can be cumbersome to write all this method chaining every time we want to accept an alert.

Reply:
Done